### ISSUE TITLE
✨ svelte-add: add `picocss`

### ISSUE THREAD

**Issue by manuel3108** [SvelteLab/SvelteLab#228]
### Describe the problem

Currently there is no possibility to add [picocss](https://github.com/svelte-add/picocss) via the svelte add command prompt.

### Describe the solution you would like

Add it :)

### Describe alternatives you have considered

It works via the command line :)

In the future we should consider to not hardcode the available adders into this project. I'm happy to provide a small api / module export from the package if necessary 👍

**Comment by paoloricciuti** [comment]
That would be fantastic...we resorted to hardcode them for two reasons actually
1. There were no api with the available adders
2. There are some adders that make no sense for the context we are using them in (for example the tauri adder makes no sense)

But I'd be super down to work together for having a more streamlined way of showing the adders

**Comment by manuel3108** [comment]
Yeah, i totally understand. At that time, the package was also not really maintained. 

Tauri and Routify i.e. make no sense at all, they won't run there for sure. What information would you need?
* Adder display name
* Adder system name (for the npx command)

I'm unsure how we should go about the icons. Currently svelte-add does not have specific icons for each adder like you guys have. I don't think we would currently be able to provide those. 

Is there any easy way to determine if node is running inside a webcontainer? Something like `process.env.IS_WEBCONTAINER` or something? With that we could filter the adders on our side, and only provide the meaningful adders

**Comment by paoloricciuti** [comment]
> Yeah, i totally understand. At that time, the package was also not really maintained.
> 
> Tauri and Routify i.e. make no sense at all, they won't run there for sure. What information would you need?
> 
> * Adder display name
> * Adder system name (for the npx command)
> 
> I'm unsure how we should go about the icons. Currently svelte-add does not have specific icons for each adder like you guys have. I don't think we would currently be able to provide those.
> 
> Is there any easy way to determine if node is running inside a webcontainer? Something like `process.env.IS_WEBCONTAINER` or something? With that we could filter the adders on our side, and only provide the meaningful adders

Yeah the icons could be a problem, we could decide for a specific iconset that has all but that would feel pretty unfair towards the other iconsets. At the end of the day we could mantain an icon map ourself and choose based on the system name. That way we would still need to mantain that but at least the user will get all the available adders just without the icon.

I don't recall if there a way to detect if you are running inside a webcontainer but we can easily pass a flag to signal that. We could also just mantain a blacklist of unuseful adders and filter with that.

**Comment by paoloricciuti** [comment]
> I would like to do the detection on our side, so that we directly decide on our side if a new adder would work inside the webcontainer.
> 
> I even found a way of detecting if we are in a webcontainer, so that's not a problem at all: `import { isWebContainer } from "@webcontainer/env";`
> 
> Do you need this as npx command of some sort, or would an export from the svelte-add package be enough?

The export would actually be better imho...but than you wouldn't be able to detect the webcontainer since the package will not run in the webcontainer instance. However again, it could easily be a function parameter

**Comment by SarcevicAntonio** [comment]
I'd say we can handle an icon map and filter stuff we feel like doesn't make sense ourselves, that would make it less complicated and coupled, right? 😄

**Comment by manuel3108** [comment]
I had a deeper look into this today and implemented it. 

As of now, we can completely ignore the `isWebContainer` stuff. In example the tauri adder has some checks in place, so that it should only be able to be used, once `cargo` and `rust` are installed. As this is probably not the case in `sveltelab` that should not be a problem.

You just need to `npm i svelte-add` and then you can do something like the following.

```js
import {  getPublicAdderListInfos } from "svelte-add"

await getPublicAdderListInfos({
	kitProject: true   // important, as svelte-add also works for non-kit projects and not every adder works everywhere
})
```

Here is also a small example project :) https://www.sveltelab.dev/8p8rkkb9i78ezyk

As for the icons: I have added the emoji svelte-add uses to generate the readmes. As far as I can tell, those appear to be default os-icons. But I think they don't fit well into this project :)
